rpcs3/rpcs3qt/gs_frame.cpp

```diff
@@ -145,6 +150,17 @@ void gs_frame::show()
 	});
 }
 
+#ifdef VK_USE_PLATFORM_WAYLAND_KHR
+std::pair<wl_display*, wl_surface*> gs_frame::wayland_handle() const
+{
```

Comment:
Move this into gs_frame::handle(). The checks in VKGSRender to check if wayland handle is valid can also be done there.